### ISSUE TITLE
How to program

### ISSUE THREAD

**Issue by tkurbad** [kamencesc/eprom2flash#1]
Hi,

thanks for the design!

More a question than an issue: How do I program the flash chip once it's soldered to the board? Can I just use a regular Epromer with correct pinout and program the device like a 27C800? Or do I have to adjust voltages etc.?

Thank you in advance,
Torsten

**Comment by kamencesc** [comment]
Original design, info and wiki can be found at sanni's github:

https://github.com/sanni/cartreader

I hasn't tested the programer yet. 27c800 is a 8bit eprom so it would be possible to program using 27c322 adapter¿? (remember that this one is 16bit!)... I don't have interest on this kind of eprom cause I allredy have a programer, I think that it's beetter to use a flash memory like 29f800 that can be programed and erased with the programer, and then use a kind of adapter to a DIP, and the cost of the memory is nearly 1€ on chinese websites.

Consider to visit original project and its wiki for more information.

**Comment by kamencesc** [comment]
Untested, but it's designed for read only.

I think that the flash memories need 5V for programming when epom needs 12V but I'm not sure, never read the datasheet for programming...

**Comment by tkurbad** [comment]
Thanks. I've had a glance at the datasheet and think the flash can handle up to 13V for programming, but I'll try with lower voltages first.
The reset and write enable lines might be a problem, though...

**Comment by kamencesc** [comment]
Yes, /reset and /we are on high for default and, I don't know if it will work with the dual bank...
